webservices/src/main/java/com/javaschool/webservices/model/ShippingRabbitMessages.java

```diff
@@ -0,0 +1,16 @@
+package com.javaschool.webservices.model;
+
+public enum ShippingRabbitMessages {
```

Comment:
What you are doing here, can be also accomplished by creating a factory method in `ShippingRabbitRPCMessage`, why did you choose to make it this way?

Reply:
I made a refactor, now the enum has a factory method.